AppCreationScripts/apps.json

```diff
@@ -48,7 +48,9 @@
 		      "replaceTokens": {
              "appId": "your_client_id",
              "tenantId": "Enter_the_Tenant_Name_Here",
```

Comment:
I believe this should be "Enter_the_Tenant_Info_Here"

Reply:
The parameters.json file has "Enter_the_Tenant_Name_Here" so if we are to change it we will have to change that too. I think now that we are moving to downloading already configured code, from what I understand, is only keys should be consistent and value can change depending on code. It will just replace that string with the tenantId. @archieag can confirm ?

Reply:
Ok that makes sense. Thanks for clarifying.

Reply:
@abhidnya13 That's right! Keys are constant, value is dependant on your code.